assets/content/cookbook/Abstract types/abstracts-with-type-params.md

```diff
@@ -0,0 +1,158 @@
+[tags]: / "abstract-type,type-params,extern"
+
+# Strict typing for stringly-typed extern code
+
+A common pattern, often found in (but not only in) dynamic languages is to denote the type of some object
```

Comment:
I'm terrible with commas but I think if you put one after "pattern" you have to put one after "dynamic languages" as well.